cms/src/services/elastic-search/common/schemas/model.js

```diff
@@ -4,12 +4,11 @@ import { ModelSchema } from '../../../../schemas/model';
 import elasticClient from '../client';
 
 const index = process.env.ES_INDEX;
-const type = process.env.ES_TYPE;
 
-ModelSchema.plugin(mongooseElasticsearch, {
+ModelSchema.plugin(mongooseElasticsearch.v7, {
   client: elasticClient,
   index,
-  type,
+  type: '_doc',
```

Comment:
this is the default `type` now for Elasticsearch, and each index can only have 1 type, so hard-coding to `"_doc"` to follow convention